addons/sale_project/views/sale_order_views.xml

```diff
@@ -19,8 +19,15 @@
                 <button class="oe_stat_button" name="action_view_milestone" type="object" icon="fa-check-square-o" invisible="not is_product_milestone or not project_ids or state == 'draft'" groups="project.group_project_milestone">
                     <field name="milestone_count" widget="statinfo" string="Milestones"/>
                 </button>
-                <button type="object" name="action_view_task" class="oe_stat_button" icon="fa-tasks" invisible="not show_task_button" groups="project.group_project_user">
-                    <field name="tasks_count" widget="statinfo" string="Tasks"/>
+                <button type="object" name="action_view_task" class="oe_stat_button" icon="fa-check" invisible="not show_task_button" groups="project.group_project_user">
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_text">Tasks</span>
+                        <span class="o_stat_value">
+                            <field name="closed_task_count"/> / 
+                            <field name="tasks_count"/>
```

Comment:
I am wondering if it should not be better to have the both fields on the same line to avoid having a space at the end of the line. 🤔 
```suggestion
                            <field name="closed_task_count"/> / <field name="tasks_count"/>
```